.codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+    status:
+        # Make all cov less fatal initially
+        project:
+            default:
+                if_no_uploads: success
+                if_not_found: success
+        patch:
+            default:
+                if_no_uploads: success
+                if_not_found: success
+        changes:
+            default:
+                if_no_uploads: success
+                if_not_found: success
+
+comment:
+    layout: "diff"
```

Comment:
It is better than it may seem as it will only post one comment and keep updating it on pushes instead of placing more comments. I intentionally reduced the layout to minimal useful detail (small text-only table with % stats).